Enh/assign-user (#12)
callbellchat/callbell-public-docs: docs/api_reference/contacts_api/patch_contacts.md

| `tags`          | string[] | A list of comma-separated values (e.g `['Call back', 'Interested']`)          |
| `custom_fields` | string{} | An object with the custom fields (e.g. `{'Billing Address': 'Main Street 1}`) |
| `name`          | string   | The name of the contact                                                       |

> | `assigned_user` | String   | Email of the user that you want to assign to a contact                        |

Reviewer: ```suggestion
| `assigned_user` | String   | Email of the the collaborator that you want to assign to a contact                        |
```

> | `unassign_user` | Boolean  | `true` is you want to remone the assigned user from a contact                 |

Reviewer: ```suggestion
| `unassign_user` | Boolean  | `true` is you want to remove the assigned collaborator from a contact                 |
```

:::caution

> When passing `custom_fields` or `tags` make sure that they already exist in your account. See [tags](https://dash.callbell.eu/settings/tags) and [custom_fields](https://dash.callbell.eu/settings/custom_fields) in your settings.

Reviewer: ```suggestion
Ensure that custom_fields and tags already exist in your account before passing them. Visit [tags](https://dash.callbell.eu/settings/tags) and [custom_fields](https://dash.callbell.eu/settings/custom_fields) in your settings for more information.
```

> Same applies for `assigned_user`, must be a valid email of a user in your account.

Reviewer: ```suggestion
Similarly, for assigned_user, use a valid email address associated with a user in your account.
```
